pystac/collection.py

```diff
@@ -710,6 +710,77 @@ def from_dict(
 
         return collection
 
+    @classmethod
+    def from_items(
+        cls: type[Collection],
+        items: Iterable[Item] | pystac.ItemCollection,
+        *,
+        id: str | None = None,
+        strategy: HrefLayoutStrategy | None = None,
+    ) -> Collection:
+        """Create a :class:`Collection` from iterable of items or an
+        :class:`~pystac.ItemCollection`.
+
+        Will try to pull collection attributes from
+        :attr:`~pystac.ItemCollection.extra_fields` and items when possible.
+
+        Args:
+            items : Iterable of :class:`~pystac.Item` instances to include in the
+                :class:`Collection`. This can be a :class:`~pystac.ItemCollection`.
+            id : Identifier for the collection. If not set, must be available on the
+                items and they must all match.
+            strategy : The layout strategy to use for setting the
+                HREFs of the catalog child objections and items.
```

Comment:
```suggestion
                HREFs of the catalog child objects and items.
```

Reply:
ah hahhahahah not sure how that snuck in.